testdata/scripts/generate_go_json_tag_postproc_disabled.txt

```diff
@@ -0,0 +1,110 @@
+gunk generate -v echo.gunk
```

Comment:
why not use a single script with two subdirectories, and one single `gunk generate ./...` call? if we had one `_disabled` and `_enabled` script for each feature, we'd have twice as many scripts.

Also, you don't need to compare the entire output to a golden file. You could grep for specific things you want or don't want, like `grep 'protobuf:".*whatever"' output.go`. With that method, you could probably avoid 80 lines in each test case.